src/team3player/DesignSchoolRobot.java

```diff
@@ -1,15 +1,29 @@
 package team3player;
 import battlecode.common.*;
 class DesignSchoolRobot extends RobotFramework {
-    MapLocation hqLoc;
+    MapLocation dsLoc;
+    int numOfLandscapers = 0;
     
     DesignSchoolRobot(RobotController rc_) {
         //super(rc_) calls the constructor of the parent class which just saves rc
         //the parent class also has the old utility functions like tryMove which need rc
         super(rc_);
         //on robot creation/start up code goes here
+        System.out.println("Design School initialization");
+        dsLoc = rc.getLocation();
+        System.out.println("I'm located at " +dsLoc.x + ":" + dsLoc.y);
     }
     public void myTurn()throws GameActionException {
         //logic to be run on every turn goes here
+        System.out.println("Design School doing stuff");
+        waitforcooldown();
+        for (Direction dir : directions) {
+            if (numOfLandscapers < 5) {
```

Comment:
Instead of a number that in the code hat is compared against (known as magic numbers in the industry) let’s make a final int variable in the class members to determine how many landscaper there should be. Like int “public final int landscaperMax = 5;” and at this line of code we have if ( numOfLandscapers < landscaperMax) it makes the code a little more declarative and readable. Especially on big code bases.